src/test/resources/run/session.yml

```diff
@@ -12,20 +12,39 @@ access_policy:
    - CREATOR
 
 services:
-   - name: service
-     image_name: service_image
-     mrenclaves: [de0249accdbeffd75cb8f8351e654ed01b8819877d6d665d0ec5d58386c264b7]
-     command: "java -jar /app/app.jar"
-     environment:
-         SCONE_MODE: hw
-         JAVA_TOOL_OPTIONS: "-Xmx256m"
-         LD_LIBRARY_PATH: "/usr/lib/jvm/java-11-openjdk/lib/server:/usr/lib/jvm/java-11-openjdk/lib:/usr/lib/jvm/java-11-openjdk/../lib"
-     pwd: /
+  
+  - name: 'pre-compute'
+    image_name: 'pre-compute-image'
+    mrenclaves: ['a45df00514ba45be07d322afb33d55cb4183adfc4e174098c365491f3af073e2']
+    pwd: '/'
+    command: 'java -jar /app/app.jar'
+    environment:
+      LD_LIBRARY_PATH: '/usr/lib/jvm/java-11-openjdk/lib/server:/usr/lib/jvm/java-11-openjdk/lib:/usr/lib/jvm/java-11-openjdk/../lib'
+      JAVA_TOOL_OPTIONS: '-Xmx256m'
+      IEXEC_TASK_ID: 'taskId'
```

Comment:
Do we need these IEXEC_* for local tests?

Reply:
No, they are not mandatory. If values are not set up, a runtime error (Dataset download failed for e.g.) will be thrown. But with those values, we can test the full workflow.